src/db.rs

```diff
@@ -71,6 +71,61 @@ impl DB {
         Ok(())
     }
 
+    pub fn migrate<P: AsRef<Path>>(&mut self, path: P) -> Result<()> {
+        if !self.dirs.is_empty() {
+            return Err(anyhow!(
+                "To prevent conflicts, you can only migrate from z with an empty \
+                 zoxide database!"
+            ));
+        }
+
+        let zdata = File::open(path)?;
+        let reader = BufReader::new(zdata);
+
+        for line in reader.lines() {
+            let line = line?;
+
+            let first_pipe = line
```

Comment:
I think it's better if these are named in a way that makes clear they're just a position within `line`.